config/default.yml

```diff
@@ -2032,6 +2032,12 @@ Lint/LambdaWithoutLiteralBlock:
   Enabled: pending
   VersionAdded: '1.8'
 
+Lint/LiteralArrayInRegexp:
```

Comment:
Won't `ArrayLiteral` be a more natural name than `LiteralArray` here?

Reply:
Yeah, true. I've updated the name.